### PR TITLE
Add command line parsing

### PR DESCRIPTION
**Pull request by gboer**
I'm in the process of adding some extra command line options to allow me to ignore when an extension is already installed. Adding a command line parser seemed to be a first good step ;)

### CODE REVIEW

File: Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
```
Comment (SLaks) [inline Properties/AssemblyInfo.cs:1]: Why did you add this?

Comment (gboer) [inline Properties/AssemblyInfo.cs:1]: It is needed for the command line parser. It reads out assembly information to display the name, copyright information etc. If you don't add this, the command line parser goes into an infinite loop actually (found that out because I didn't have it at first)
